into/backends/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def sc():
+    pyspark = pytest.importorskip('pyspark')
+    return pyspark.SparkContext('local', 'into-test')
+
+
+@pytest.fixture(scope='session')
+def sqlctx(sc):
+    sparksql = pytest.importorskip('pyspark.sql')
+    return sparksql.SQLContext(sc)
```

Comment:
This tells me that it's probably possible to convert a `PySparkRDD` to a `SchemaRDD/spark.DataFrame` which makes me pretty happy.